ui/src/app/job-details/tasks/tasks.component.ts

```diff
@@ -21,6 +22,10 @@ import {TaskMetadata} from '../../shared/model/TaskMetadata';
 })
 export class TaskDetailsComponent implements OnInit, OnChanges {
   @Input() tasks: TaskMetadata[] = [];
+  @Input() jobid: string;
+  parentJobId = '';
```

Comment:
I don't really understand why you need a `parentJobId` when its just a mirror of `jobId`. Aren't they set equal on init and whenever jobId changes? I might be missing something but it seems you could do with just `jobId`.